### PR TITLE
Ariel

### PR DESCRIPTION
**Pull request by gleason9113**
Okay, I've collected all the various updates (except for Cera's most recent fixes and Alex's new tests) into this PR.  I'd like to merge this into main, then I think that we should be able to merge Cera's in, which should fix the bug Alex found, I think?

I also added better fonts.

### CODE REVIEW

**Review by sashetov — CHANGES_REQUESTED** [review]
ariel, there's some issue with your latest PR. i checked out your branch and ran the commands to build it and got:
/home/sasheto/data/psu/cs-465/final-proj/final-project/server/index.js:50
<<<<<<< HEAD
^^

SyntaxError: Unexpected token '<<'

if you fix that I'll approve

**Comment by sashetov** [comment]
also, in general, things with backend will go more smothely if im the only one modifying that file

File: server/index.js [hunk 1]
```diff
@@ -59,6 +60,20 @@ app.get('/join', (req, res) => {
...
+            // first player goes first
+            isYourTurn: true,
+            opponentName: players[player2_socket.id].playerName
+          },
+        };
+        let data2 = {
+          status: 'ok',
+          msg: 'you have an opponent',
+          data: {
+>>>>>>> main
```
Comment (sashetov) [inline server/index.js]: maybe just use the original file

**Comment by gleason9113** [comment]
Should be fixed.  At least, the IDE says so.  I am hopeful, but wary of being hurt again.

**Review by sashetov — APPROVED** [review]
it works this time. same issue with board not updating on first move in both tabs but I would merge

**Pull request merged**